service-proxy.js

```diff
@@ -51,6 +52,9 @@ var RATE_LIMIT_KILLSWITCH = 'killswitch';
 
 var CN_HEADER_BUFFER = new Buffer('cn');
 
+// TODO: ring change trigger
```

Comment:
We need this before going global, we can test on canaries without ringpop change trigger.